components/graph/git-graph-actions.js

```diff
@@ -95,7 +95,7 @@ class Reset extends ActionBase {
         context &&
         context.node() &&
         remoteRef.node() != context.node() &&
-        remoteRef.node().date < context.node().date
+        remoteRef.node().timestamp < context.node().timestamp
```

Comment:
Why do you use timestamps? Is it because numeric comparison will be a little faster?